test/python/WMCore_t/Services_t/Requests_t.py

```diff
@@ -133,7 +133,7 @@ def testRecoveryFromConnRefused(self):
         """Connections succeed after server down"""
         import socket
         self.rt.stop()
-        req = Requests.Requests(self.urlbase, {'req_cache_path': self.cache_path})
+        req = Requests.Requests(self.urlbase, {'req_cache_path': self.cache_path, 'pycurl': False})
```

Comment:
again, I'm not sure if False is appropriate default here since you're moving towards using pycurl everywhere.

Reply:
Usually you shouldn't bother reviewing the test changes. But I can answer this one too, the test below (testSecureOddPort_with_pycurl) tests with pycurl library, while this one is meant to test httplib.